### PR TITLE
Update UG (elaborate instructions to check for Java 17)

### PR DESCRIPTION
**Pull request by riccoljy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
...
+1. Ensure you have `Java 17` or above installed on your computer by performing the following:
+   1. Open your system terminal
+      1. Windows: <br>
+         1. Hold down Windows button and R (⊞ Win + R ) 
+         2. Type `cmd` and press enter
+         
+      1. MacOS -- do one of the following: <br>
+          - Click the Launchpad icon in the Dock, type `Terminal` in the search field, then click Terminal, OR
+          - In the Finder, open the `/Applications/Utilities` folder, then double-click Terminal.
+   1. Type `java -version` and press enter
```
Comment (tingxuanp) [inline docs/UserGuide.md:28]: Is this a typo? Supposed to be 2.?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
...
 ## Quick start
 
-1. Ensure you have `Java 17` or above installed in your Computer. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
+1. Ensure you have `Java 17` or above installed on your computer by performing the following:
+   1. Open your system terminal
+      1. Windows: <br>
+         1. Hold down Windows button and R (⊞ Win + R ) 
+         2. Type `cmd` and press enter
+         
+      1. MacOS -- do one of the following: <br>
```
Comment (tingxuanp) [inline docs/UserGuide.md:25]: Is this a typo? Supposed to be 2.?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
...
+          - Click the Launchpad icon in the Dock, type `Terminal` in the search field, then click Terminal, OR
+          - In the Finder, open the `/Applications/Utilities` folder, then double-click Terminal.
```
Comment (tingxuanp) [inline docs/UserGuide.md:27]: I really like how you included these instructions :D

File: docs/UserGuide.md [hunk 4]
```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
...
+   1. Type `java -version` and press enter
+   1. If you have `Java 17 installed, the terminal should look something like:
```
Comment (tingxuanp) [inline docs/UserGuide.md:29]: Same here is this a typo or is it supposed to be 1.?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
...
+   ```
+         C:\Users\UserName>java -version
+         java version "17.0.12" 2024-07-16 LTS
+         Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
+         Java HotSpot(TM) 64-Bit Server VM (build 17.0.12+8-LTS-286, mixed mode, sharing)
+    ```
+2. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
```
Comment (tingxuanp) [inline docs/UserGuide.md:36]: Why is this 2. and the next step also 2.?

**Review by tingxuanp — COMMENTED** [review]
LGTM just some typos I think

File: docs/UserGuide.md (hunk 5 above)
Comment (riccoljy) [reply docs/UserGuide.md:36]: image

GFMD/MD automatically numbers that's why there's "repeated" numbers. Having
```
1.
1.
1.
```
will make MD auto-number :D

Comment (tingxuanp) [reply docs/UserGuide.md:36]: Oh yea i realised after i pulled from your branch... should have done that from the start oops sorry! then LGTM

**Pull request merged**